app/__init__.py

```diff
@@ -9,6 +10,7 @@
 document_store = DocumentStore()  # noqa, has to be imported before views
 antivirus_client = AntivirusClient()  # noqa
 metrics = GDSMetrics()  # noqa
+redis_store = RedisClient()
```

Comment:
Optional: I think looking at the admin, they call this `redis_client` rather than `redis_store`. Although super minor, it might be good for consistency to have this as `redis_client = RedisClient()`

https://github.com/alphagov/notifications-admin/blob/master/app/extensions.py#L7

Reply:
Why did I change this 🤔

Reply:
Ah, I took it from notifications-api which calls it redis_store 🤷 

https://github.com/alphagov/notifications-api/blob/master/app/__init__.py#L64

Reply:
My preference is redis_client so I'll change it (esp since I'm going to make changes elsewhere in response to comments probably)